sale_edi.py

```diff
@@ -758,9 +762,17 @@ def import_sales(cls, edi_sale=None):
         for fname in files:
             if fname[-4:].lower() not in KNOWN_EXTENSIONS:
                 continue
-            with open(fname, 'r', encoding='latin-1') as fp:
-                data = fp.readlines()
-                sale_edi = cls.import_edi_file([], data)
+            for count in range(retry, -1, -1):
```

Comment:
Why do we create a loop if there's no way that it is going to iterate?

At line 775 there's a break and at 774 a return.

Reply:
> Why do we create a loop if there's no way that it is going to iterate?

Because try to read a file. Our traceback, the reason is "mount" system and try to retry some times to read the file.

Break -> skip inside the for -> continue
return -> can not import edi file